### DISCUSSION TITLE
iframe youtube embed not working

### DISCUSSION THREAD

**Discussion by usma0118** [mealie-recipes/mealie#2347] [Q&A]
I am trying to embed youtube video in mealie recipe via iframe.
iframe is saved but not rendered, looking into history this should have been fixed in #2009 

running version: v1.0.0beta-5 
Build: ee121a12f8db33ecb4db5f8582f7ea9788d019e4

**Comment by hay-kot** [comment]
> running version: v1.0.0beta-5
> Build: https://github.com/hay-kot/mealie/commit/ee121a12f8db33ecb4db5f8582f7ea9788d019e4

Does not contain the code from #2009 - You'd have to use the nightly image to get this to work.
